scripts/run_tests_fire.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env sh
+
+set -xeu
+
+CWD=$(pwd)
+
+unzip_all() {
```

Comment:
do we ever call this function ?

Reply:
@vfdev-5 No we don't. Let me remove this